<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
background-color: white;
}</string>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>751</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView {
   background-color: transparent;
	border:1px solid #EFEFEF;
}

</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="redBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#redBtn{
background:white;
border:1px solid rgb(255, 84, 92);
border-radius:5px;
padding:2px 4px;
color:rgb(255, 84, 92);
}

QPushButton#redBtn:checked{
background:rgb(255, 84, 92);
color:white;
}</string>
    </property>
    <property name="text">
     <string>red</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="blueBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#blueBtn{
background:white;
border:1px solid #3288E8;
border-radius:5px;
padding:2px 4px;
color:#3288E8;
}

QPushButton#blueBtn:checked{
background:#3288E8;
color:white;
}</string>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#yellowBtn{
background:white;
border:1px solid rgb(255, 226, 6);
border-radius:5px;
padding:2px 4px;
color:rgb(255, 226, 6);
}

QPushButton#yellowBtn:checked{
background:rgb(255, 226, 6);
color:white;
}</string>
    </property>
    <property name="text">
     <string>yellow</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="allBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#allBtn{
background:white;
border:1px solid gray;
border-radius:5px;
padding:2px 4px;
color:gray;
}

QPushButton#allBtn:checked{
background:gray;
color:white;
}</string>
    </property>
    <property name="text">
     <string>all</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="setRedBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#setRedBtn{
border:1px solid rgb(255, 84, 92);
border-radius:5px;
padding:2px 4px;
background:rgb(255, 84, 92);
color:white;
}</string>
    </property>
    <property name="text">
     <string>set red</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
